darts/models/kalman.py

```diff
@@ -0,0 +1,81 @@
+import numpy as np
+import pandas as pd
+
+from abc import ABC
+
+from typing import Optional
+from filterpy.kalman import KalmanFilter
+
+from .smoothing_model import SmoothingModel
+from ..timeseries import TimeSeries
+
+
+class Kalman(SmoothingModel, ABC):
+
+    def __init__(
+            self, 
+            dim_x: int = 1, 
+            dim_z: int = 1, 
+            x = np.array([[1.]]),
+            F = np.array([[1]]),
```

Comment:
This could be None by default, and initialised to a default matrix of appropriate dimensions (depending on `dim_x`). Similar for H and P.

Reply:
sure for dim_x. H and P have the "state" dimensions not "measure" dimensions. I will make a comment about it, however in real-life kf should be passed with a custom configuration. I will mention that in doc string.